README.md

```diff
@@ -67,3 +67,27 @@ Which will load the following `<script>` asynchronously:
 ```
 
 View the [full list of supported script parameters](https://developer.paypal.com/docs/checkout/reference/customize-sdk/#script-parameters).
+
+### Using a CDN
+
+The paypal-js script is also available on the [UNPKG CDN](https://unpkg.com/). The paypal.browser.js build assigns the `loadScript` function to the window object as `window.paypalLoadScript`. Here's an example:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <script src="https://unpkg.com/@paypal/paypal-js/dist/paypal.browser.min.js"></script>
```

Comment:
Note the absence of the version number here. This is done on purpose so this code snippet always uses the latest version from UNPKG. There's redirect functionality provided by UNPKG when you leave out the version number. It redirects to the latest version.

<img width="1485" alt="Screen Shot 2020-08-31 at 11 30 24 AM" src="https://user-images.githubusercontent.com/534034/91743545-987d2880-eb7d-11ea-8590-293f622691fc.png">

Reply:
@gregjopa I like the simplicity of this approach and using what's already available to us.  How is their CDN distributed?  And can we configure a CNAME to map to this IP to customize the url?

Reply:
> How many edge servers to they have and where?

I was wondering that too. The [About section on the website](https://unpkg.com/) mentions the unpkg CDN is powered by Cloudflare Enterprise and Cloudflare sponsor's the work. I found this talk from the author that goes into more detail: https://youtu.be/2rhkgB8Cohc?t=1737

> And can we configure a CNAME to map to this IP to customize the url?

I imagine we could but I'm unsure it would be worth the effort. The majority of developers will be using this module via npm and bundling it with their own assets.

Reply:
Oh, right!  Good point!